src/components/ActionBar/index.tsx

```diff
@@ -8,12 +8,12 @@ const SPACING = 20
 
 const StyledDiv = styled.div<{ rank: number }>`
   align-items: center;
-  background: ${({ theme }) => theme.colorsDeprecated.white};
-  border: 1px solid ${({ theme }) => theme.colorsDeprecated.gray300};
+  background: ${({ theme }) => theme.colors.neutral.backgroundWeak};
+  border: 1px solid ${({ theme }) => theme.colors.neutral.borderWeak};
```

Comment:
Your PR made me notice there was a problem with border colors 😄 

I made fix right here: https://github.com/scaleway/scaleway-ui/pull/978

Reply:
I guess no action to take here (Other than review and validate the PR).

Reply:
Sorry I made another change for neutral color, you should put border as:

```js
border: 1px solid ${({ theme }) => theme.colors.neutral.border};
```

Reply:
done